setup.py

```diff
@@ -48,15 +50,14 @@ def read(fname):
     },
 
     install_requires=[
-        'pyarrow==12.0.0',
+        'pyarrow>=12.0.0',
         'fastparquet==2023.4.0',
```

Comment:
Similar question here as for GAS for fastqparquet. Should we change this from `==` to `>=`?

Reply:
Updated in f539df7